.github/ISSUE_TEMPLATE/new_issue.yaml

```
@@ -0,0 +1,66 @@
+name: New Issue
+description: Report something that doesn't fit into above categories
+labels:
+  - Something else
```

I now realise that you had misunderstood my previous comment. I had meant to suggestion that you use "question" instead of "Something else", not "Something else?" with a question mark:
```suggestion
  - question
```

---

Should we add a question mark emoji too?

---

Sure, why not.

---

Probably in front